stacker_blueprints/util.py

```diff
@@ -0,0 +1,17 @@
+def check_properties(properties, property_list, resource):
```

Comment:
Maybe change `property_list` to `allowed_properties` or `available_properties` to make it a little more self-explanatory and differentiate it from properties some more.